### ISSUE TITLE
Change clip start

### ISSUE THREAD

**Issue by felagund** [ozmartian/vidcutter#51]
Once I click "set clip start", I no longer seem able to change it, nor by dragging, nor by clicking it again since it gets greyed out. But what if I make a mistake and want to chagne the clip start?

**Comment by felagund** [comment]
Oh, and once I click "clip end", I cannot change that either. But I would often like to do that - especially now that we have the "continue" option after the media is saved.

**Comment by MacSass** [comment]
I would agree - being new to vidcutter that was something I was looking for and somehow expecting.
The ability to "update" an clip start or end without having to delete and recreated it from scratch would be a great addtion for me.
